File: platform/sdk/python/examples/graph/graph-029_ac15_f13_lag_spike.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.ac15F13LagSpike(15)
if not result["passed"] or not result["recovered"]:
    sys.exit(1)
print("XJ-OK: graph-029_ac15_f13_lag_spike")
sys.exit(0)

File: platform/sdk/nodejs/examples/graph/graph-029_ac15_f13_lag_spike.js
const { GraphClient } = require("../../mox-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.ac15F13LagSpike(15);
if (!result.passed || !result.recovered) process.exit(1);
console.log("XJ-OK: graph-029_ac15_f13_lag_spike");
process.exit(0);
